package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "staker",
+  "version": "1.0.0",
```

Comment:
Let's bump this version number to 1.0.1. We will cut a new release off main which includes these changes sometime after it is merged.

Reply:
Fix: https://github.com/withtally/staker/pull/121/commits/608048830f2872aeb20835fd1cc8edd031ddfe14